boards/arm/thingy91_nrf9160/thingy91_nrf9160_common.dts

```diff
@@ -84,6 +84,8 @@
 		pwm-led2 = &pwm_led2;
 		rgb-pwm = &pwm0;
 		nmos-pwm = &pwm2;
+		mcuboot-button0 = &button0; // shouldn't be 6th Pin?
```

Comment:
TBC

Reply:
@jhn-nordic This assignment configures the serial recovery trigger gpio-pin. Should this be GPIO0 pin6 as it results from (legacy)Kconfig in mcuboot?

Reply:
On Thingy:91 it is P0.26 on the nRF9160 that should trigger serial recovery if pressed on boot